### PR TITLE
Wording and command of install guide / "LIBVIRTD_ARGS="--listen"" leads to failure with libvirt6.0.0

### PR DESCRIPTION
**Pull request by Hudratronium**
Hi,
made some minor changes regarding wording. Also added some notes as this part of documentations aims for an productive environment.

With regards

### CODE REVIEW

File: source/installguide/hypervisor/kvm.rst [hunk 1]
```diff
@@ -289,15 +289,7 @@ cloudstack-agent and should already be installed.
...
-   On RHEL or CentOS modify ``/etc/sysconfig/libvirtd``:
-
-   Uncomment the following line:
-
-   .. parsed-literal::
-
-      #LIBVIRTD_ARGS="--listen"
-
-   On Ubuntu 14.04: modify ``/etc/default/libvirt-bin``
+      On Ubuntu 14.04: modify ``/etc/default/libvirt-bin``
```
Comment (GabrielBrascher) [inline source/installguide/hypervisor/kvm.rst:292]: It would also be good to remove Ubuntu 14.04 from documentation as it is no longer supported.

**Review by GabrielBrascher — COMMENTED** [review]
Thanks for the proposed changes. I have raised some suggestions.

**Review by Hudratronium — COMMENTED** [review]
inserted changes as suggested - thanks for your support

**Review by DaanHoogland — APPROVED** [review]

**Comment by DaanHoogland** [comment]
requesting docbuild

**Review by GabrielBrascher — APPROVED** [review]

**Comment by rhtyd** [comment]
@Hudratronium that's no true, you need to mask the services to get old behaviour. We've fixed that in cloudstack-agent I think, @shwstppr @davidjumani - can you confirm?

On Ubuntu we've to do this to then get libvirtd listen on tcp/ssl port:
```
systemctl mask libvirtd.socket libvirtd-ro.socket libvirtd-admin.socket libvirtd-tls.socket libvirtd-tcp.socket
systemctl restart libvirtd
```

**Comment by Hudratronium** [comment]
> 
> 
> @Hudratronium that's no true, you need to mask the services to get old behaviour. We've fixed that in cloudstack-agent I think, @shwstppr @davidjumani - can you confirm?
> 
> On Ubuntu we've to do this to then get libvirtd listen on tcp/ssl port:
> 
> ```
> systemctl mask libvirtd.socket libvirtd-ro.socket libvirtd-admin.socket libvirtd-tls.socket libvirtd-tcp.socket
> systemctl restart libvirtd
> ```

Hi @rhtyd 

At that time i raised an issue that, as far as i got this, is fixed for release 4.15.1
https://github.com/apache/cloudstack/issues/4754

On Ubuntu - as fas as i understood @shwstppr , that was already part of the configuration script.

**Comment by rhtyd** [comment]
@Hudratronium yes that's correct. This is fixed for 4.15.1 in https://github.com/apache/cloudstack/pull/4756 and therefore I think the doc changes are not necessary, this PR can be closed. What do you think?

**Comment by Hudratronium** [comment]
@rhtyd 
okay, will do so :-)

**Pull request closed without merging**